pages/docs/v3/updating/3-0.md

```diff
@@ -270,3 +270,27 @@ Capacitor 3 supports Android 11 (API 30), so you can update your SDK target to 3
      androidxCoreVersion =  '1.2.0'
      androidxMaterialVersion =  '1.1.0-rc02'
 ```
+
+### Remove unused and redundant permissions
+
+Depending on which plugins you are using, you can optionally remove unused permissions from your app's `AndroidManifest.xml` file. [The manifest in new Capacitor apps](https://github.com/ionic-team/capacitor/blob/main/android-template/app/src/main/AndroidManifest.xml) only includes `INTERNET` because permissions are now meant to be added when plugins are installed. Follow these steps to remove unused permissions:
```

Comment:
```suggestion
Depending on which plugins you are using, you can optionally remove unused permissions from your app's `AndroidManifest.xml` file. [The manifest in new Capacitor apps](https://github.com/ionic-team/capacitor/blob/main/android-template/app/src/main/AndroidManifest.xml) only includes `INTERNET` because most permissions are now meant to be added automatically when plugins are installed. Follow these steps to remove unused permissions:
```

Reply:
It seems that many permissions are added manually, though.

Reply:
Ah sorry I think I partly misunderstood this part. Yeah it makes sense.